src/js/views/Projects/HelpDialog.jsx

```diff
@@ -24,7 +25,7 @@ function HelpDialog({ onClose }) {
         />
         <h1 className="my-4 font-bold">{t('projects.searchHelpFields')}</h1>
         <ul className="list-disc list-inside max-h-36 ml-5 font-mono overflow-scroll">
-          {fields.map((field) => {
+          {fields.sort(byValue((f) => f, byString())).map((field) => {
```

Comment:
I think this can be simplified to just `fields.sort(byString())` since you're passing the value unchanged to `byString`, as the [discriminator](https://sort-es.netlify.app/by-value#discriminator) here is the identity function.

`byValue` would be useful if `fields` were objects and you had to sort on a property like:

```javascript
fields.sort(byValue((f) => f.name, byString()))
```